lib/create-space-api.ts

```diff
@@ -552,19 +552,20 @@ export default function createSpaceApi(makeRequest: MakeRequest) {
      * const contentful = require('contentful-management')
      *
      * client.getSpace('<space_id>')
-     * .then((space) => space.getSpaceTeams({'limit': 100}))
-     * .then((spaceTeamCollection) => console.log(spaceTeamCollection))
+     * .then((space) => space.getTeams())
+     * .then((teamsCollection) => console.log(teamsCollection))
      * .catch(console.error)
      * ```
      */
-    getSpaceTeams(query: QueryOptions = {}) {
```

Comment:
why did we remove the query params here?

Reply:
I didn't define them in the app-SDK at all. Since I have this difference between the app-sdk and the actual implementation, I wanted to remove this malicious implementation ASAP. As [Kado mentioned](https://contentful.slack.com/archives/C0201KJQQD9/p1623930046041100), we should look out for those cases where we define the method without having the implementations ready.
I saw that `space.getUsers` also doesn't allow parameters, so I thought it's fine to stick to this solution for now until we enhance it.

Reply:
But from a raw cma `sdk` perspective we should add support for query options. I can imagine using the `sdk` in a different code base where I want pagination support. We should not tie the implementation of the cma sdk to the `app-sdk`.

Reply:
As discussed, I'm readding the query params :)